Formula/rustscan.rb

```diff
@@ -0,0 +1,22 @@
+class Rustscan < Formula
+  desc "Faster Nmap Scanning with Rust"
+  homepage "https://github.com/rustscan/rustscan"
+  url "https://github.com/RustScan/RustScan/archive/1.3.tar.gz"
+  version "1.3.0"
+  sha256 "3bbaf188fa4014a57596c4d4f928b75bdf42c058220424ae46b94f3a36b61f81"
+  license "MIT"
+  depends_on "rust" => :build
+  depends_on "nmap" => :optional
+  def install
+    system "cargo", "install", *std_cargo_args
+    # bin.install "target/release/rustscan"
+  end
+
+  test do
+    # Due to law, users installing this package may not be able to portscan websites
```

Comment:
Will it fail if you scan something like 0.0.0.0?

Reply:
Nope! Do you want me to add that as a test? :)

Reply:
It's a result we can be pretty sure of. I think it would be a good fit.